vendor/user_profile.ps1.default

```diff
@@ -23,3 +23,20 @@
 
 ## <Continue to add your own>
 
+# Delete default powershell aliases that conflict with bash commands
+# If you prefer the powershell aliases remove this.
+if (get-command git) {
```

Comment:
should we really do that by default?

Reply:
It's only on first run and the user can turn it off if they want.  I feel like anyone that types `curl` in a shell expects to be using `curl` not `invoke-webrequest`.  I know it posses me off.  :-)

We could comment themand let the user uncomment them.

Reply:
Hrm yeah I know the problem, just not too sure about changing defaults for users

Reply:
rectified